features/netsocket/nsapi_dns.cpp

```diff
@@ -31,6 +31,9 @@
 #include "PlatformMutex.h"
 #include "SingletonPtr.h"
 
+using namespace std::chrono;
+using rtos::Kernel::Clock;
```

Comment:
Is this necessary?

Reply:
No, but it saves typing `rtos::Kernel` a fair few times in the file. Could just go halfway with the more conventional `using namespace rtos`.